management/management.go

```diff
@@ -46,6 +46,39 @@ func NewService(uri string, a auth.IAuthenticator) (*Service, error) {
 	return &m, nil
 }
 
+// NewInsecureTLSService creates a new Service instance using the provided
```

Comment:
There is no tests for the new code in management.go. I just noticed that there is an equivalent test for this in `management_test.go`, for example:  TestService_NewService()

Reply:
That's a good point. This actually brought up a bigger issue that management tests weren't being run as part of CI. I've fixed this and added missing tests.